README.md

```diff
@@ -63,6 +62,21 @@ redis = { version = "0.27.5", features = ["tokio-comp"] }
 redis = { version = "0.27.5", features = ["async-std-comp"] }
 ```
 
+## Connection Pooling
+
+When using a sync connection, it is recommended to use a connection pool in order to handle
+disconnects or multi-threaded usage. This can be done using the `r2d2` feature.
+
+```
+redis = { version = "0.27.5", features = ["r2d2"] }
+```
+
+For async connections, connection pooling isn't necessary. The `MultiplexedConnection` is
```

Comment:
It might be worth mentioning here a caveat that if blocking commands are used, connection pooling is still potentially worthwhile since other requests will be blocked as well.

Reply:
done